src/deploy/hosting/prepare.js

```diff
@@ -71,7 +71,7 @@ module.exports = function (context, options) {
 
     const data = {
       config: convertConfig(cfg),
-      labels: deploymentTool.labels,
+      labels: deploymentTool.labels(),
```

Comment:
Why was this turned into a function?

Reply:
Previously, deploymentTool was setting module.exports to an object, which allowed us to use a getter function for labels & value. I wanted to move it over to use `export function` syntax to be more consistent with the rest of the codebase, but in order to do so, I had to turn this from a getter function into a normal one